syspagen/syspagen.c

```diff
@@ -51,11 +61,14 @@ typedef struct {
 
 
 struct {
-	addr_t pkernel;
-	off_t offs;
-	size_t maxsz;
+	int type;
+	syspage32_t *syspage32;
+	syspage64_t *syspage64;
+
+	unsigned long pkernel;
+	unsigned long offs;
```

Comment:
I guess you want to always have these 64-bit. long is 64 bit only on some targets
```suggestion
	unsigned long long pkernel;
	unsigned long long offs;
```

Reply:
Thx, done